dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/validation/MetadataValidation.java

```diff
@@ -96,6 +98,14 @@ public List<ErrorRest> validate(SubmissionService submissionService, InProgressS
                             "/" + WorkspaceItemRestRepository.OPERATION_PATH_SECTIONS + "/" + config.getId() + "/" +
                                 input.getFieldName());
                     }
+                    if ("local.hasCMDI".equals(fieldName)) {
```

Comment:
Could we use a variable? Possibly configurable

Reply:
Added constant: https://github.com/dataquest-dev/DSpace/blob/feature/se-9-upload-CMDI/dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/validation/MetadataValidation.java#L47